controllers/status.go

```diff
@@ -555,6 +556,7 @@ func mapCSV(csv *unstructured.Unstructured) operatorsv1.StatusCondition {
 		phase, _ := statusCondition["phase"].(string)
 		message, _ := statusCondition["message"].(string)
 		reason, _ := statusCondition["reason"].(string)
+		conditionType, _ := statusCondition["type"].(string)
```

Comment:
Have you been able to test this on a running cluster? For `mapCSV` you're grabbing the type field from the csv status condition, but I don't think such a field exists. Is it able to populate?
```
jagray@MacBook-Pro backplane-operator % oc explain csv.status.conditions
KIND:     ClusterServiceVersion
VERSION:  operators.coreos.com/v1alpha1

RESOURCE: conditions <[]Object>

DESCRIPTION:
     List of conditions, a history of state transitions

     Conditions appear in the status as a record of state transitions on the
     ClusterServiceVersion

FIELDS:
   lastTransitionTime	<string>
     Last time the status transitioned from one status to another.

   lastUpdateTime	<string>
     Last time we updated the status

   message	<string>
     A human readable message indicating details about why the
     ClusterServiceVersion is in this condition.

   phase	<string>
     Condition of the ClusterServiceVersion

   reason	<string>
     A brief CamelCase message indicating details about why the
     ClusterServiceVersion is in this state. e.g. 'RequirementsNotMet'
```

Reply:
Commenting here for posterity - We discussed over slack I was testing incorrectly as I was testing on an old image. I am opening new PR with new change we decided on